km_api/know_me/admin.py

```diff
@@ -71,12 +71,15 @@ class KMUserAccessor(admin.ModelAdmin):
         'km_user',
         'user_with_access',
         'accepted',
-        'can_write_everywhere')
+        'can_write',
+        'has_private_profile_access')
```

Comment:
I think it's worth keeping these separate in case our needs change later. It's simple enough to control this from the client app.